zk_toolbox/crates/zk_inception/src/commands/args/run_server.rs

```diff
@@ -14,4 +15,6 @@ pub struct RunServerArgs {
     #[clap(long, short)]
     #[arg(trailing_var_arg = true, allow_hyphen_values = true, hide = false, help = MSG_SERVER_ADDITIONAL_ARGS_HELP)]
     additional_args: Vec<String>,
+    #[clap(long, help = MSG_SERVER_BUILD_HELP)]
+    pub build: bool,
```

Comment:
why do we need it? 
just cargo build normally enough

Reply:
The user could need to build the server before running it, in order to e.g. prepare for a CI run or just to have it ready for running. Therefore `zk_inception server --build` is useful. Although I'm now realizing that the code that actually built the server is not there, I'll add it again.

Reply:
Do we need it? 
I'm not strongly against but probably it's a bit of redundant

Reply:
We do, otherwise revert tests fail due to timeout waiting for server to build.